lib/jsdom/level2/style.js

```diff
@@ -173,6 +173,13 @@ core.HTMLStyleElement._init = function() {
 
     evaluateStylesheet.call(this, content, this.sheet, this._ownerDocument.URL);
   });
+
+  this.addEventListener('DOMNodeRemovedFromDocument', function() {
```

Comment:
So, if you edit level1/core.js's definition of the Document prototype, you can add a _descendantRemoved method, which receives (parent, child) as arguments. Just be sure to call the super-class `Node.prototype._descendantRemoved.apply(this, arguments)`.